docs/reference/ingest/apis/put-pipeline.asciidoc

```diff
@@ -76,7 +76,53 @@ Processors run sequentially in the order specified.
 `version`::
 (Optional, integer)
 Version number used by external systems to track ingest pipelines.
+
```

Comment:
@jrodewig, would you mind looking over these additions to the ingest pipeline docs?